### PR TITLE
Add EG4ll BMS to Dev

### PR DESCRIPTION
**Pull request by Justice12t**
Working on getting this added to Dev.

[Checklist](https://louisvdw.github.io/dbus-serialbattery/general/supported-bms#add-by-opening-a-pull-request)

  - [ ] Add your battery class and battery class import in alphabetical order in the [`etc/dbus-serialbattery/dbus-serialbattery.py`](https://github.com/Louisvdw/dbus-serialbattery/blob/dev/etc/dbus-serialbattery/dbus-serialbattery.py)
  - [x] Add your BMS to the [BMS feature comparison](../general/features#bms-feature-comparison) page by editing [`docs/docs/general/features.md`](https://github.com/Louisvdw/dbus-serialbattery/blob/dev/docs/docs/general/features.md)
  - [x] Add your BMS to the [Supported BMS](../general/supported-bms) page by editing [`docs/docs/general/supported-bms.md`](https://github.com/Louisvdw/dbus-serialbattery/blob/dev/docs/docs/general/supported-bms.md)
  - [ ] Do not import wildcards `*`
  - [ ] If available populate `self.max_battery_charge_current` and `self.max_battery_discharge_current` with values read from the BMS
  - [ ] If available populate `self.unique_identifier` with a unique value to distinguish the BMS in a multiple battery setup
  - [ ] If your BMS don't run with the default settings add installation notes to the [How to install, update, disable, enable and uninstall](../general/install#bms-specific-settings) [`docs/docs/general/install.md`](https://github.com/Louisvdw/dbus-serialbattery/blob/dev/docs/docs/general/install.md)
  - [ ] If your BMS needs custom settings that the user should be able to change, add it below the `; --------- BMS specific settings ---------` section in the [`etc/dbus-serialbattery/config.default.ini`](https://github.com/Louisvdw/dbus-serialbattery/blob/dev/etc/dbus-serialbattery/config.default.ini)
  - [ ] Make sure the GitHub Actions run fine in your repository. In order to make the GitHub Actions run please select in your repository settings under `Actions` -> `General` -> `Actions permissions` the option `Allow all actions and reusable workflows`. Check also in your repository settings under `Actions` -> `General` -> `Workflow permissions` if `Read and write permissions` are selected. After this changes you have to make a new commit, if you don't see any Actions run. This will check your code for Flake8 and Black Lint errors. [Here](https://py-vscode.readthedocs.io/en/latest/files/linting.html) is a short instruction on how to set up Flake8 and Black Lint checks in VS Code. This will save you a lot of time.

### CODE REVIEW

File: docs/docs/general/features.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ If the `MAX_CELL_VOLTAGE` \* `cell count` is reached for `MAX_VOLTAGE_TIME_SEC`
 
 ## BMS feature comparison
 
-| Feature                                                                                  | Ant   | Daly  | ECS                | Heltec | HLPdataBMS4S | JK BMS | Life/Tian Power | LLT/JBD | MNB <sup>(1)</sup> | Renogy | Seplos | Sinowealth <sup>(1)</sup> |
+| Feature                                                                                  | Ant   | Daly  | ECS                | Heltec | HLPdataBMS4S | JK BMS | Life/Tian Power/EG4ll | LLT/JBD | MNB <sup>(1)</sup> | Renogy | Seplos | Sinowealth <sup>(1)</sup> |
```
Comment (mr-manuel) [inline docs/docs/general/features.md:124]: You are adding a new BMS not extending a existing one.

**Comment by mr-manuel** [comment]
I added a checklist that helps you to get ready for merge. Also compare your BMS class with https://github.com/Louisvdw/dbus-serialbattery/blob/dev/etc/dbus-serialbattery/bms/battery_template.py, since your class seems to be from a older template.

**Comment by Justice12t** [comment]
sorry, I'm sure you can tell this isn't an area I excel in.  I'll continue to fix the errors in the coming days.

**Comment by mr-manuel** [comment]
@Justice12t any updates?

**Pull request closed without merging**